src/certifier.cc

```diff
@@ -371,6 +371,10 @@ bool certifier::framework::policy_store::Deserialize(string& in) {
   return true;
 }
 
+int certifier::framework::policy_store::get_num_signed_claims() {
```

Comment:
This interface was missing previously. Swig-bundling complains ... so it was good to find and fix this missing piece, as a side-benefit of trying out this Swig business.